src/main/kotlin/io/github/mojira/arisa/modules/FutureVersionModule.kt

```diff
@@ -21,10 +21,10 @@ class FutureVersionModule(
             val latestVersion = project.versions.lastOrNull(::isFutureVersion.complement())
             assertNotNull(latestVersion).bind()
 
-            latestVersion!!.add().toFailedModuleEither().bind()
-            tryRunAll(removeFutureVersions).bind()
-            addComment(CommentOptions(message)).toFailedModuleEither().bind()
-            resolveAsAwaitingResponse().toFailedModuleEither().bind()
+            latestVersion!!.add()
+            removeFutureVersions.forEach(::run)
+            addComment(CommentOptions(message))
```

Comment:
add comment should be always after resolving, and it should only happen if the module didnt fail to do the resolve